cups/snmp.c

```diff
@@ -1355,7 +1359,7 @@ asn1_get_packed(
     unsigned char **buffer,		/* IO - Pointer in buffer */
     unsigned char *bufend)		/* I  - End of buffer */
 {
-  int	value;				/* Value */
+  unsigned	value;			/* Value */
```

Comment:
Again, no need for this change.

Reply:
Please see my previous explanation. Example byte sequence for my proof of concept is `b'\x1F\xFF\xFF\xFF\xFF\xFF'`.